lnt/tests/kv_engine_testsuites.conf

```diff
@@ -3,5 +3,10 @@
         "server-name": "kv_engine",
         "client-name": "kv_engine",
         "server-db-key": "KV"
+    },
+    {
+        "server-name": "kv_engine_mh",
```

Comment:
"mh" might not be that obvious to an end-user - can we spell it out long "mad-hatter" when it's shown to in the UI?

Reply:
Sure, probably even better to change this to "kv_engine_mad-hatter" so we don't need any specific UI conversion